src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala

```diff
@@ -0,0 +1,53 @@
+package org.apache.james.gatling.jmap
+
+import java.util.UUID
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import org.apache.james.gatling.control.UserFeeder
+import org.apache.james.gatling.utils.RandomStringGenerator
+
+import scala.util.Random
+
+case class MessageId(id: String = RandomStringGenerator.randomString)
+case class RecipientAddress(address: String)
+case class Subject(subject: String = RandomStringGenerator.randomString)
+case class TextBody(text: String = RandomStringGenerator.randomString)
+
+object JmapMessages {
+
+  def sendMessages(messageId: MessageId, recipientAddress: RecipientAddress, subject: Subject, textBody: TextBody) =
+    JmapAuthentication.authenticatedQuery("sendMessages", "/jmap")
+      .body(StringBody(
+        s"""[[
+          "setMessages",
+          {
+            "create": {
+              "${messageId.id}" : {
+                "from": {"name":"$${username}", "email": "$${username}"},
+                "to":  [{"name":"${recipientAddress.address}", "email": "${recipientAddress.address}"}],
+                "textBody": "${textBody.text}",
+                "subject": "${subject.subject}",
+                "mailboxIds": ["$${outboxMailboxId}"]
+              }
+            }
+          },
+          "#0"
+          ]]"""))
+      .check(status.is(200))
```

Comment:
I don't see how I can combine two jmap checks, so I wonder if it add any benefit (at least for the status, that is easily readable....)